static/templates/tables/exportWidget.hbs

```diff
@@ -1,7 +1,12 @@
 <div class="results-info results-info--simple">
   <h1 class="results-info__title">{{this}} results</h1>
-  <button type="button"
-    class="js-export-widget button button--primary-contrast button--export results-info__download">
-      Export this data
-  </button>
+  <div class="results-info__download js-tooltip-container">
+    <button type="button"
+      class="js-export button button--primary-contrast button--export">
+        Export this data
+    </button>
+    <div id="export-tooltip" role="tooltip" class="tooltip tooltip--under tooltip__content" aria-hidden="true">
+      <span>Exports are limited to 100,000 records— add filters to narrow results, or export bigger data sets with <a href="http://fec.gov/data/DataCatalog.do?cf=downloadable" target="_blank">FEC bulk data exporter</a>.</span>
```

Comment:
Can we pass `DOWNLOAD_CAP` to the template so we don't have to hard-code the number here?